src/components/item-related-purchase/styled.js

```diff
@@ -0,0 +1,56 @@
+import styled from 'styled-components';
+
+const LatoBold = 'Lato Bold';
+
+export const LinkWrapper = styled.a`
```

Comment:
since we are using react router we can use a Link component that they expose and recommends